### PR TITLE
feat: upgrade to sdk 13 beta and compiler 7

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

**Comment by marc0olo** [comment]
is this backwards compatible when an old compiler version is used? (which can be configured, right?)

**Comment by thepiwo** [comment]
@marc0olo haven't tried, but I wouldn't guess so, backwards/forwards compatibility is not really supported in aeproject anyways

**Comment by thepiwo** [comment]
waiting for a final sdk v13 release before being ready with this PR

**Comment by thepiwo** [comment]
@davidyuk @marc0olo this is now ready for review

File: src/init/artifacts/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "test": "mocha ./test/**/*.js --timeout 0 --exit"
   },
   "dependencies": {
-    "@aeternity/aepp-sdk": "^12.1.3"
+    "@aeternity/aepp-sdk": "^13.0.0-beta.0"
```
Comment (kenodressel) [inline src/init/artifacts/package.json:9]: why still beta here?

**Review by marc0olo — APPROVED** [review]
@thepiwo sorry for the confusion. with correct usage of the linked package it works 👍 

```sh
===== Starting Tests =====

> aeproject-project@1.0.0 test
> mocha ./test/**/*.js --timeout 0 --exit



  ExampleContract
    ✔ ExampleContract: set and get (256ms)
    ✔ ExampleContract: get undefined when not set before


  2 passing (2s)
  ```

**Review by kenodressel — APPROVED** [review]

**Pull request merged**